df_to_azure/export.py

```diff
@@ -138,15 +138,15 @@ def upload_to_blob(table):
     create_dir(tmp_path)
 
     table.df.to_csv(
-        full_path_to_file, index=False, sep="^", quotechar='"'
+        full_path_to_file, index=False, sep="^", quotechar='"', line_terminator=os.linesep
     )  # export file to staging
 
     logging.info(f"start uploading blob {table.name}...")
     with open(full_path_to_file, "rb") as data:
         blob_client.upload_blob(data, overwrite=True)
     logging.info(f"finished uploading blob {table.name}!")
 
-    os.remove(full_path_to_file)
+    # os.remove(full_path_to_file)
```

Comment:
@erfannariman  dit is denk ik niet de bedoeling? was waarschijnlijk voor debugging.